src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpsServer.java

```diff
@@ -25,80 +25,83 @@
-    protected HttpsServer () {
+    protected HttpsServer() {
     }
 
     /**
-     * creates a HttpsServer instance which is initially not bound to any local address/port.
-     * The HttpsServer is acquired from the currently installed {@link HttpServerProvider}
-     * The server must be bound using {@link #bind(InetSocketAddress,int)} before it can be used.
-     * The server must also have a HttpsConfigurator established with {@link #setHttpsConfigurator(HttpsConfigurator)}
-     * @throws IOException
+     * Creates a {@code HttpsServer} instance which is initially not bound to any
+     * local address/port. The {@code HttpsServer} is acquired from the currently
+     * installed {@link HttpServerProvider}. The server must be bound using
+     * {@link #bind(InetSocketAddress,int)} before it can be used. The server
+     * must also have a {@code HttpsConfigurator} established with
+     * {@link #setHttpsConfigurator(HttpsConfigurator)}.
+     *
+     * @throws IOException if an I/O error occurs
      */
-    public static HttpsServer create () throws IOException {
-        return create (null, 0);
+    public static HttpsServer create() throws IOException {
+        return create(null, 0);
     }
 
     /**
-     * Create a <code>HttpsServer</code> instance which will bind to the
-     * specified {@link java.net.InetSocketAddress} (IP address and port number)
+     * Create a {@code HttpsServer} instance which will bind to the specified
+     * {@link java.net.InetSocketAddress} (IP address and port number).
      *
      * A maximum backlog can also be specified. This is the maximum number of
-     * queued incoming connections to allow on the listening socket.
-     * Queued TCP connections exceeding this limit may be rejected by the TCP implementation.
-     * The HttpsServer is acquired from the currently installed {@link HttpServerProvider}
-     * The server must have a HttpsConfigurator established with {@link #setHttpsConfigurator(HttpsConfigurator)}
+     * queued incoming connections to allow on the listening socket. Queued TCP
+     * connections exceeding this limit may be rejected by the TCP implementation.
+     * The {@code HttpsServer} is acquired from the currently installed
+     * {@link HttpServerProvider}. The server must have a {@code HttpsConfigurator}
+     * established with {@link #setHttpsConfigurator(HttpsConfigurator)}.
      *
-     * @param addr the address to listen on, if <code>null</code> then bind() must be called
-     *  to set the address
-     * @param backlog the socket backlog. If this value is less than or equal to zero,
-     *          then a system default value is used.
+     * @param addr the address to listen on, if {@code null} then
+     *             {@link #bind(InetSocketAddress,int)}  must be called to set
+     *             the address
+     * @param backlog the socket backlog. If this value is less than or equal to
+     *               zero, then a system default value is used
```

Comment:
There is more than one sentence here, and the second sentence is a full sentence so I believe you'll need a full-stop at the end.

Reply:
Punctuation added. You can view the change in commit: https://github.com/openjdk/jdk/pull/610/commits/ef5b9fa75886007484f15d74ddf2529f4054ee66